src/XRP/issued-currency-client.ts

```diff
@@ -192,6 +202,33 @@ export default class IssuedCurrencyClient {
     }
   }
 
+  /**
+   * Subscribes to an account's incoming transactions, and triggers a callback upon recieving each transaction.
+   * @see https://xrpl.org/monitor-incoming-payments-with-websocket.html
+   * @see https://xrpl.org/subscribe.html
+   *
+   * @param account The account from which to subscribe to incoming transactions, encoded as an X-Address.
+   * @param callback The function to trigger upon recieving each transaction from the ledger.
+   * @returns The response from the websocket confirming the subscription.
+   */
+  public async monitorIncomingPayments(
```

Comment:
again, I think we need to be clear on what we're monitoring here... if this is subscribing to all transactions that affect the account, that might involve outgoing payments, as well as other transaction types that modify the account settings, trust lines, or other objects.  We should probably call this `monitorAccountTransactions` or something more generic.

Reply:
unless we want to get fancy and provide our own callback that takes a callback, where our callback first filters things for payments-only and then iff the transaction was a payment executes the user's callback.  I think that could be achieved with like... currying or some sort of functional fanciness?  I don't think we need to do that though, I think we're better off just naming this better